test/Function/StackArgsWithFormals.js

```diff
@@ -0,0 +1,141 @@
+//Test1 : Tests Optimization with bailout.
+var shouldBailout = false;

```

Comment:
Done.

---
In reply to: [67928173](https://github.com/Microsoft/ChakraCore/pull/1135#discussion_r67928173) [](ancestors = 67928173,67812487,67765923)